android/logto-client/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -21,6 +25,96 @@ open class LogtoClient(
 
     fun isAuthenticated() = idToken != null
 
+    fun getAccessToken(callback: RetrieveCallback<AccessToken>) =
+        getAccessToken(null, null, callback)
+
+    fun getAccessToken(
+        resource: String?,
+        scope: List<String>?,
+        callback: RetrieveCallback<AccessToken>,
+    ) {
+        if (!isAuthenticated()) {
```

Comment:
would implement `isAuthenticated` as a getter better?

Reply:
Need to change the convention.
If the convention is changed, this code will be updated in another PR.

Reply:
if we don't perform network requests for this, a getter will be better

Reply:
It will be done in LOG-1450